tools/buildf2c

```diff
@@ -70,11 +70,7 @@ tar -xvf f2c.tar
 
 gunzip -rf f2c/*
 
-cd f2c
```

Comment:
I'm not convinced we should have this file in here at all. Again, it looks like it's specifically for OSX. Is there really no better way to install it there than using this script? I guess having an untested and possibly slightly broken script is better than none?

Reply:
Maybe it could be added to homebrew? I see it in macPorts.

Reply:
Well I certainly won't do that, but it would be great if someone else would. In the meantime leaving this unmaintained script here is less work than fixing the problem.

Reply:
I'd do it if it wasn't so annoying. No versioned link to the files AFAICT, and brew's not going to like that. Barely even versions (dates, I think).